tests/controller/elementControllerTest.js

```diff
@@ -154,7 +166,7 @@ before(async function () {
         return doc._id;
     });
     idElement2 = await Element.create({
-        idQR: idQR2,
```

Comment:
Is this idElement1 ?

Reply:
Not sure to understand

Reply:
you never asigned idElement1 but it is used later for soft/hard dlete tests